x/hard/client/cli/query.go

```diff
@@ -90,35 +89,46 @@ func queryModAccountsCmd(queryRoute string, cdc *codec.Codec) *cobra.Command {
 		),
 		Args: cobra.NoArgs,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			cliCtx := context.NewCLIContext().WithCodec(cdc)
-
-			name := viper.GetString(flagName)
-			page := viper.GetInt(flags.FlagPage)
-			limit := viper.GetInt(flags.FlagLimit)
+			clientCtx, err := client.GetClientTxContext(cmd)
```

Comment:
```suggestion
			clientCtx, err := client.GetClientQueryContext(cmd)
```
And same for the other queries. I just fix this on one of my PRs 😄